doc/user/content/self-managed-deployments/_index.md

```diff
@@ -0,0 +1,353 @@
+
+## Workflow
+
+The following outlines the workflow process, summarizing how the various
+components work together:
+
+1. **Install the Helm chart**: This deploys the Materialize Operator to your
+   Kubernetes cluster.
+
+1. **Create a Materialize instance**: Apply a Materialize custom resource. The
+   operator detects this and creates all necessary Kubernetes resources,
+   including the `environmentd`, `balancerd`, and `console` pods.
+
+1. **Connect to the instance**: Use the Materialize Console on port 8080 to
+   connecto to the `console` service endpoint or SQL client on port 6875 to
+   connect to the `balancerd` service endpoint.
+
+   If authentication is enabled, you must first connect to the Materialize
+   Console and set up users.
+
+1. **Create clusters**: Issue SQL commands to create clusters. Materialize
+   coordinates with the operator to provision StatefulSets for replicas.
+
+1. **Run your workloads**: Create sources, materialized views, indexes, and
+   sinks on your clusters.
+
+## Terraform Modules
+
+To help you get started, Materialize provides Terraform modules.
+
+{{< important >}}
+These modules are intended for evaluation/demonstration purposes and for serving
+as a template when building your own production deployment. The modules should
+not be directly relied upon for production deployments: **future releases of the
+modules will contain breaking changes.** Instead, to use as a starting point for
+your own production deployment, either:
+
+- Fork the repo and pin to a specific version; or
+
+- Use the code as a reference when developing your own deployment.
+
+{{</ important >}}
+
+{{< tabs >}}
+{{< tab "Unified Terraform Modules  (New!)" >}}
+### Unified Terraform Modules
+
+Materialize provides a [**unified Terraform
+module**](https://github.com/MaterializeInc/materialize-terraform-self-managed/tree/main?tab=readme-ov-file#materialize-self-managed-terraform-modules),
+which provides concrete examples and an opinionated model for deploying Materialize.
+
+{{< yaml-table data="self_managed/terraform_list" >}}
+
+{{< /tab >}}
+{{< tab "Legacy Terraform Modules" >}}
+### Legacy Terraform Modules
+
+{{< yaml-table data="self_managed/terraform_list_legacy" >}}
+{{< /tab >}}
+{{< /tabs >}}
+
+## Relationship to Materialize concepts
+
+Self-managed deployments implement the same core Materialize concepts as the Cloud offering:
+
+- [**Clusters**](/concepts/clusters/): Identical behavior, but backed by Kubernetes StatefulSets
+- [**Sources**](/concepts/sources/): Same functionality for ingesting data
+- [**Views**](/concepts/views/): Same query semantics and incremental maintenance
+- [**Indexes**](/concepts/indexes/): Same in-memory query acceleration
+- [**Sinks**](/concepts/sinks/): Same data egress capabilities
+
+The Self-Managed deployment model adds the Kubernetes infrastructure layer (Helm
+chart and operator) but does not change how you interact with Materialize at the
+SQL level.
```

Comment:
It might be better to just have something at the top of the page saying something like
```
Self-managed deployments use same concepts and deployment mechanisms as Materialize's Cloud offering.
```

It might be worth while adding the clusters concepts link to the "clusters and replicas" section, then having a list of "other materialize concepts". I don't think we need to call out that everything is the same on a per item basis.